setup.cfg

```diff
@@ -0,0 +1,4 @@
+
+[sqla_testing]
+requirement_cls=databricks.sqlalchemy.dialect.requirements:Requirements
+profile_file=profiles.txt
```

Comment:
It's because `sqla_testing` isn't a feature of pytest but of the custom pytest plugin written by SQLAlchemy. It _only_ reads from `setup.cfg` and doesn't know about pyproject.toml

Reply:
In a follow-up PR (adding SQLAlchemy 2.0 compatibility) I'm going to reorganise the dialect files so that setup.cfg sits in the SQLAlchemy dialect directory rather than here at the top-level.